advanced-usage/index.md

```diff
@@ -219,6 +219,7 @@ The current options include
     - clicking on this allows selecting which `NAMED_VALUE_FLOAT/INT` MAVLink message to track, out of
       any that have been received so far (including custom ones)
     - it is also possible to specify a display unit, a value multiplier, and an icon
+    - several pre-made templates are available for usage with common variables
```

Comment:
Can this be moved to the line above, so that the text about the customisation options is directly above the image for customisation?